releasenotes/sf/README.md

```diff
@@ -18,6 +18,22 @@ If you installed `sf` with `npm`, run `npm install @salesforce/cli@latest-rc -g`
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
+* NEW: Get JSON output from these Salesforce Functions commands with the `--json | -j` flag:
+
+    * `sf deploy functions`
+    * `sf env create compute`
+    * `sf env delete`
+    * `sf env logdrain add`
+    * `sf env logdrain remove`
+    * `sf env var set`
+    * `sf env var unset`
+    * `sf logout functions`
+    * `sf run function`
```

Comment:
* `sf whoami functions`

Reply:
Actually, I won't include `sf whoami functions` in the list of commands that just got the `--json` flag because the command already has it.  See the [CLI Reference](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_whoami_commands_unified.htm#cli_reference_whoami_functions_unified) (which, as of July 18, hasn't been updated with last week's RC info).   By comparison, docs for `env var set` [don't yet show that flag](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_env_commands_unified.htm#cli_reference_env_var_set_unified).